src/ai/backend/agent/docker/agent.py

```diff
@@ -434,6 +434,15 @@ async def apply_network(self, cluster_info: ClusterInfo) -> None:
                         },
                     }
                 )
+        else:
+            if "alternate_bridge" in self.local_config["container"]:
```

Comment:
We can merge these lines to reduce the indentation level.

Reply:
Thank you. I applied it